packages/sit-onyx/src/components/OnyxAvatar/types.ts

```diff
@@ -2,13 +2,26 @@ import type { IconSize, IconSizeProp } from "../OnyxIcon/types";
 
 export type OnyxAvatarProps = IconSizeProp<Exclude<IconSize, "12px">> & {
   /**
-   * Label (e.g. the user name), required for accessibility.
-   * Will also determine the displayed initials.
+   * Username. Will determine the displayed initials.
    */
-  label: string;
+  username: DisplayName;
```

Comment:
(Optional): When I read "username" I am always thinking "login name" e.g., `joca96` or `baracko`, so I would prefer something like:
```suggestion
  displayName: DisplayName;
```
```suggestion
  fullName: DisplayName;
```

Reply:
Currently this is aligned with the UserMenu. Would you also change it there too?

Reply:
Sounds good :)

Reply:
But you don't have to change it